packages/asc-ui/src/components/Tag/TagStyle.ts

```diff
@@ -0,0 +1,30 @@
+import styled, { css, styledComponents, Theme } from '@datapunt/asc-core'
+import { readableColor } from 'polished'
+import { Props as TypographyProps } from '../Typography/TypographyStyle'
+import Typography from '../Typography'
+import { color as getColor } from '../../utils'
+
+export type Props = {
+  backgroundColor?: keyof Theme.Tint | keyof Theme.TypeLevel
```

Comment:
backgroundColor indicates that you can just set a color code. Perhaps change it to variant? and have a enum set of variants that will link to a limited set of colors?

Reply:
btw, we also have to do this with our Button component...

Reply:
Changed to `color` for now to even it with the `Button`. However `colorType` might be a better fitl, but results in breaking changes...